examples/all-clusters-app/esp32/main/DeviceCallbacks.cpp

```diff
@@ -69,10 +71,16 @@ void DeviceCallbacks::DeviceEventCallback(const ChipDeviceEvent * event, intptr_
             // connectivity. MDNS still wants to refresh its listening interfaces to include the
             // newly selected address.
             chip::app::Mdns::StartServer();
+#ifdef CONFIG_RENDEZVOUS_MODE_BLE
+            if(event->CHIPoBLESubscribe.ConId)
```

Comment:
What does this check indicate?

Reply:
It returns the BLE connection object, if there is a BLE connection else it will return NULL.